src/Sylius/Component/Order/Model/OrderItem.php

```diff
@@ -27,6 +27,8 @@ class OrderItem implements OrderItemInterface
 
     protected int $unitPrice = 0;
 
+    protected int $originalPrice = 0;
```

Comment:
if we will decide to go with this solution, IMO original price should be null by default, on `channelPricing` we can have originalPrice equal null so here should be same